config/locales/en.yml

```diff
@@ -163,6 +163,10 @@ en:
   membership_applications:
     new:
       title: Apply for a Membership
+      instructions: some instructions here. 3 main sections. explain importance of the uploaded documents
```

Comment:
To process your application, we need information about you and in which category you are applying as well as documents proving your skills in this/those categories.